.github/workflows/run_tests.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install flake8 pep517
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements-test.txt ]; then pip install -r requirements-test.txt; fi
```

Comment:
nit: I see we also miss I would prefer if requirements-test.txt would contain `-r requirements.txt` as the first line (that's the common convention I saw in the other projects).

Then I would re-write these two lines like that:
```
        if [ -f requirements-test.txt ]; then
          pip install -r requirements-test.txt
        elif [ -f requirements.txt ]; then
          pip install -r requirements.txt
        fi
```

Reply:
Done